ChangeLog.2013

```diff
@@ -2468,7 +2468,7 @@
 
 2013-03-31 John Ralls 
 
-	* Add libgncmod-ledger-core to satsify dependency in libgncmod-gnome.
+	* Add libgncmod-ledger-core to satisify dependency in libgncmod-gnome.
```

Comment:
satisfy

Reply:
Serves me right for trying to fix something manually instead of letting my browser correct the spelling. 6b612b75ee9bb4f94c2292a8c155a2150d168f2f